rust/sdk/src/service/events.rs

```diff
@@ -402,8 +394,7 @@ pub struct SubscribeMonotonicRequest {
     /// previously interrupted stream. In this case, StartFrom::Offsets is used,
     /// otherwise StartFrom::Snapshot indicates that the PondService shall figure
     /// out where best to start out from, possibly sending a `State` message first.
-    #[serde(flatten)]
-    pub from: StartFrom,
+    pub from: OffsetMap,
```

Comment:
please add a test that contains the previously written serialisation format and verifies that the new serialisation format is identical

Reply:
There actually is a test that caught my error:
https://github.com/Actyx/Actyx/actions/runs/6801666115/job/18493040522?pr=583#step:3:485